src/_imagingft.c

```diff
@@ -719,7 +727,7 @@ bounding_box_and_anchors(FT_Face face, const char *anchor, int horizontal_dir, G
 static PyObject *
 font_getsize(FontObject *self, PyObject *args) {
     int width, height, x_offset, y_offset;
-    int load_flags;               /* FreeType load_flags parameter */
+    int load_flags; /* FreeType load_flags parameter */
     int error;
```

Comment:
```suggestion
    int error;
    int load_flags;               /* FreeType load_flags parameter */
```
[If the indentation is a concern](https://github.com/python-pillow/Pillow/pull/8015/files#r1579447074), then this could be a solution.

Reply:
Hmm, every variable in this function is declared at the top at the moment, and I think the consistency of that is good.

I'm happy just leaving it as it currently is in this PR.